.envrc.example

```
@@ -0,0 +1,19 @@
+toplevel="$(git rev-parse --show-toplevel)"
+
+# ensure the pre-commit git hook is in place
+pushd $toplevel/.git/hooks >/dev/null &&
+    ln -sf ../../.githooks/pre-commit &&
+    popd >/dev/null
+
+# build the managed go sdk
+pushd $toplevel >/dev/null &&
+    make go-check &&
+    popd >/dev/null
+
+# detect os and architecture on darwin and linux
+machine="$(go env GOOS)-$(uname -m)"
+
+# export GOROOT and PATH for managed go sdk
+GOROOT="$toplevel/.build/$machine/go/$(cat .go-version)"
+PATH="$GOROOT/bin:$PATH"
```

There is already a makefile target that prints a modified path with the correct toolchain directory added (no need to repeat the same magic the makefile does to determine the path):
```suggestion
PATH="$(make -C "$toplevel" go-bin-path)"
```

---

This needs `--no-print-directory` to work.